lib/app/router.js

```diff
@@ -23,7 +23,8 @@ function recursiveRoutes(routes, tab, components) {
 var _components = []
 var _routes = recursiveRoutes(router.routes, '\t\t', _components)
 uniqBy(_components, '_name').forEach((route) => { %>
-const <%= route._name %> = () => import('<%= route.component %>' /* webpackChunkName: "pages/<%= route.name %>" */)
+// const <%= route._name %> = () => import('<%= route.component %>' /* webpackChunkName: "pages/<%= route.name %>" */)
```

Comment:
I did this update because i not figure out how i can make this line work when the file have `.ts` and not `.js`. The error look like:

```bash
 error  in ./.nuxt/router.ts

Syntax Error: Unexpected token (5:24)

  3 | import Router from 'vue-router';
  4 | Vue.use(Router);
> 5 | const _36d8919f = () => ;
    |                         ^
  6 | ('/frontend/pages/index.vue');
  7 | const scrollBehavior = (to, from, savedPosition) => {
  8 |     if (savedPosition) {



 @ ./.nuxt/index.ts 32:0-40
 @ ./.nuxt/client.ts
 @ multi webpack-hot-middleware/client?reload=true ./.nuxt/client.ts
```